source/api/presentation/2.1/index.md

```diff
@@ -1159,6 +1159,7 @@ Segments of both static images and canvases may be selected by adding a [rectang
 }
 ```
 
+<a name="image-api-selection"/>
```

Comment:
For anchors IDs in Kramdown, this can just be `{: #image-api-selection}` on the line directly above the paragraph (or any block)